tests/test-ansible-functional.sh

```diff
@@ -94,7 +94,7 @@ function gate_job_exit_tasks {
   # which was present when the trap was initiated.
   # This would be the success/failure of the test.
   export TEST_EXIT_CODE=$?
-  source "${COMMON_TESTS_PATH}/test-log-collect.sh"
+  source "/opt/openstack-ansible/scripts/test-log-collect.sh"
```

Comment:
@hughsaunders this looks to be breaking the builds.
https://rpc.jenkins.cit.rackspace.net/job/RPC-IRR_rpc-maas-master-trusty-newton/513

/var/lib/jenkins/workspace/RPC-IRR_rpc-maas-master-trusty-newton/rcbops/rpc-maas/tests/test-ansible-functional.sh: line 97: /opt/openstack-ansible/scripts/test-log-collect.sh: No such file or directory

Reply:
Removing that commit as its not cross-branch compatible.